sdk/ai/azure-ai-projects/azure/ai/projects/telemetry/agents/_ai_agents_instrumentor.py

```diff
@@ -1645,25 +1645,25 @@ def __init__(
         self.last_message: Optional[ThreadMessage] = None
         self.instrumentor = instrumentor
 
-    def on_message_delta(self, delta: "MessageDeltaChunk") -> None:
+    def on_message_delta(self, delta: "MessageDeltaChunk", **kwargs) -> None:
```

Comment:
We shall discuss more this change, ie adding  **kwargs

When customers would use kwargs? and how we document what parameters they can read?